content/telegraf/v1.22/get_started.md

```diff
@@ -15,12 +15,12 @@ After you've [downloaded and installed Telegraf](/telegraf/v1.22/install/), you'
 
 ## Configure Telegraf
 
-Define which plugins Telegraf will use in the configuration file. Each configuration file needs at least one enabled [input plugin](/telegraf/v1.22/plugins/inputs/) (where the metrics come from) and at least one enabled [output plugin](/telegraf/v1.22/plugins/outputs/) (where the metrics go).
+Define which plugins Telegraf will use in the configuration file. Each configuration file needs at least one enabled [input plugin]((/{{< latest "telegraf" >}}/plugins/inputs/) (where the metrics come from) and at least one enabled [output plugin]((/{{< latest "telegraf" >}}/plugins/outputs/) (where the metrics go).
```

Comment:
should be (/{{< latest "telegraf" >}} - think you have an extra beginning parenthesis in a couple of places